src/helpers/ComponentRegister.ts

```diff
@@ -74,6 +61,16 @@ export class ComponentRegister {
    */
   static Link: LinkComponent = defaultComponentRegisterOptions.components.Link;
 
+  /**
+   * The {@link LinkButton} to allow this library to hook into routers or
+   * other custom linking libraries.
+   *
+   * Defaults to `"a"`. See {@link ComponentRegisterComponents.LinkButton} and
+   * {@link defaultComponentRegisterOptions}.
+   */
+  static LinkButton: LinkButtonComponent =
+    defaultComponentRegisterOptions.components.LinkButton;
```

Comment:
Given this has a nearly identical interface to `ComponentRegister.Link`, can we default to using `ComponentRegister.Link` (or a wrapper of it) if no `LinkButton` is provided in `init`? Users shouldn't need to provide this unless they need to use a different implementation.